### ISSUE TITLE
Tooltip activate/deactivate functionality (not just on hover/mouseout)

### ISSUE THREAD

**Issue by ChillyBots** [highcharts/highcharts#17982]
# Tooltip activate/deactive
Currently, tooltips can be activated on hover, however this doesn't suit many use cases. 
Ideally, tooltips should have sevreal options to control how they are activated / deactived

`activate: 'mouseover' | 'click' | 'mousedown' | 'mouseup' | 'keydown'`
There is a hack for this (enabling the classic tooltip hover when clicked) but this limits usage (you cannot open a tooltip and leave it open while moving the mouse for example)

`deactivate: 'mouseout' | 'click'`
tooltips close or refocus to another datapoint whenever the mouse is moved, however this is not always desired

With these two options, you could still accomplish current behaviour where tooltips open and close on hover of each bullet (`a:mouseover, d:mouseout`) but you could also create granular interactions where you have multiple tooltips open, or only one open on click at a time

Unfortunately, I can't create a proof of concept

----

**You can vote for this feature by adding a thumbs-up reaction to this post.**

**Comment by karolkolodziej** [comment]
@ChillyBots have you tried using the tooltip `refresh` method?
By doing so you could programmatically show the tooltip for given points- [demo](https://jsfiddle.net/BlackLabel/va37ezc6/).

**Comment by ChillyBots** [comment]
Hey @karolkolodziej, thanks for the reply - unfortunately this doesn't quite match the use case, as hovering over any other marker now displays that marker's tooltip. You cannot just click a marker and display its tooltip until you click. 

We have managed to produce something which works - however, it involves hiding all tooltips with CSS and cloning anything we've clicked on into a new element and manually placing it, removing it from the DOM when we register another click. 

To say its not nearly as good as a native solutino would be an understatement
